content/ja/handbook/leadership/1-1/index.md

```
@@ -0,0 +1,290 @@
+
+1. **積極的に聴く**。自己評価は難しく、人々はしばしばスキルセットを過大評価または過小評価します。彼らの評価をすぐに否定しないでください。共通点を見つけ、彼らの全体的な目標を理解することに集中してください。
+1. **コントロール**。会話を軌道に乗せ続けるためにコントロールを維持してください。焦点は現在のスキルセットと能力、そしてそれらを会社内でのキャリアパスに向けて育てる方法に当たるべきです。
+1. **適応**。異なるパーソナリティタイプへのアプローチを適応させることが鍵です。スキルセットを過大評価する人には、どこで期待に応え、応えられていないかの具体的な内容を提供すべきです。失敗した領域をより率直に（しかし常に思いやりを持って）指摘する必要があるかもしれません。スキルセットを過小評価する人は、ネガティブな面に集中する傾向があるため、正しくやっていることをより強調する必要があるかもしれません。また、全員が昇進したいわけではありません。現在の役割にとても満足しており、そこにとどまりたい人もいます。それもサポートすべきです。[Learning & Development Handbook](/handbook/people-group/learning-and-development/)でさらに詳しく見てください。
+
+## 1-1の移行
+
+既存のチームメンバーのマネジメントを引き継ぐとき、チームメンバーが心理的に安全な移行を体験できるよう以下のガイドラインを検討してください。
+
+### マネージャー移行ミーティング
+
+[透明性](/handbook/values/#transparency)を実現するために、マネージャーは現在のマネージャーが将来のマネージャーに提供するすべての情報が、チームメンバーの完全な見える範囲で行われることを確保することを検討すべきです。これはすべてのチームメンバーに適切とは限りません。チームメンバーがこの会話をオープンに行うことに快適かどうかを尋ね、彼らの境界を尊重すべきです。透明性を選択する場合、移行ミーティングをスケジュールして[マネージャー移行ミーティングのアジェンダ](#manager-transition-meeting-agenda-example)に従ってください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)